### PR TITLE
Example/asme 10

### PR DESCRIPTION
**Pull request by jviqueg**
Example 10 for documentation based on ASME Section VIII Division 2

### CODE REVIEW

File: ansys/dpf/core/examples/examples.py [hunk 1]
```diff
@@ -19,3 +19,4 @@
 msup_transient = os.path.join(_module_path, "msup_transient_plate1.rst")
 simple_cyclic = os.path.join(_module_path, "file_cyclic.rst")
 distributed_msup_folder = os.path.join(_module_path, 'msup_distributed')
+ASME_SecVIII_Div2 = os.path.join(_module_path, 'ASME_SecVIII_Div2.rst')
```
Comment (cbellot000) [inline ansys/dpf/core/examples/examples.py:22]: we try to not push result files in this repo to keep our package as light as possible. Could you please refer to [this ](https://github.com/pyansys/pydpf-core/pull/179#discussion_r893067628) comment?

File: examples/00-basic/10-ASME_SecVIII_Div2.py [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+# values: alfasl = 2.2 & m2 = .288
+#
+
+alfasl = input("Please introduce alfasl parameter from ASME\n")
+alfasl = float(alfasl)
+m2 = input("Please introduce m2 parameter from ASME\n")
+m2 = float(m2)
+
+###############################################################################
+# Streses & strains
```
[the remark this answers is not in the document]
Comment (jviqueg) [reply examples/00-basic/10-ASME_SecVIII_Div2.py:50]: Done

File: examples/00-basic/10-ASME_SecVIII_Div2.py [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+dataSource = model.metadata.data_sources
+
+timeScoping = dpf.Scoping()
+timeScoping.location = dpf.locations.time_freq
+timeScoping.ids = [4]
+
+
+###############################################################################
+# Parameters input
+# ~~~~~~~~~~~~~
```
[the remark this answers is not in the document]
Comment (jviqueg) [reply examples/00-basic/10-ASME_SecVIII_Div2.py:38]: Done for all lines

File: examples/00-basic/10-ASME_SecVIII_Div2.py [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+# User must go to ASME Section III Division 2 and get parameters alfasl & m2
+# If the user want to replicate same results as the workshop, please introduce these
+# values: alfasl = 2.2 & m2 = .288
+#
+
+alfasl = input("Please introduce alfasl parameter from ASME\n")
```
[the remark this answers is not in the document]
Comment (jviqueg) [reply examples/00-basic/10-ASME_SecVIII_Div2.py:44]: Done

File: ansys/dpf/core/examples/examples.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (jviqueg) [reply ansys/dpf/core/examples/examples.py:22]: Done

Comment (jviqueg) [reply ansys/dpf/core/examples/examples.py:22]: I followed Ayush example

**Review by cbellot000 — APPROVED** [review]

**Comment by jviqueg** [comment]
Hi @cbellot000 ,
All required checks are successful, but there is a warning message about line 427 of downloads.py. I reviewed it and I don't see any issue. Should I merge the branch? Please let me know. Thanks!

**Pull request merged**